src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-config-modal/rgw-config-modal.component.html

```diff
@@ -0,0 +1,253 @@
+                        <cd-helper i18n>The SSL certificate in PEM format.</cd-helper>
+                        </label>
+                        <div class="cd-col-form-input">
+                        <input type="file"
+                                formControlName="ssl_cert"
+                                (change)="fileUpload($event.target.files, 'ssl_cert')">
+                        <span class="invalid-feedback"
+                                *ngIf="configForm.showError('ssl_cert', frm, 'required')"
+                                i18n>This field is required.</span>
+                        </div>
+                    </div>
+                </div>
+    
+                <div *ngIf="configForm.getValue('encryptionType') == 'aws:kms' || configForm.getValue('encryptionType') == 'AES256'">  
+                    <div class="form-group row">
+                        <label class="cd-col-form-label"
+                            for="client_cert">
+                        <span i18n>Client Certificate</span>
+                        <cd-helper i18n>The Client certificate in PEM format.</cd-helper>
+                        </label>
+                        <div class="cd-col-form-input">
+                        <input type="file"
+                                formControlName="client_cert"
+                                (change)="fileUpload($event.target.files, 'client_cert')">
+                        <span class="invalid-feedback"
+                                *ngIf="configForm.showError('client_cert', frm, 'required')"
+                                i18n>This field is required.</span>
+                        </div>
+                    </div>
+                </div>
+    
+                <div *ngIf="configForm.getValue('encryptionType') == 'aws:kms' || configForm.getValue('encryptionType') == 'AES256'">  
+                    <div class="form-group row">
+                        <label class="cd-col-form-label"
+                            for="client_key">
+                        <span i18n>Client Private Key</span>
+                        <cd-helper i18n>The Client Private Key in PEM format.</cd-helper>
+                        </label>
+                        <div class="cd-col-form-input">
+                        <input type="file"
+                                (change)="fileUpload($event.target.files, 'client_key')">
+                        <span class="invalid-feedback"
+                                *ngIf="configForm.showError('client_key', frm, 'required')"
+                                i18n>This field is required.</span>
+                        </div>
+                    </div>
+                </div>
+  
+        <div class="modal-footer">
+          <cd-form-button-panel (submitActionEvent)="onSubmit()"
+                                [submitText]="actionLabels.SUBMIT"
+                                [form]="configForm"></cd-form-button-panel>
+        </div>
+        </div>
```

Comment:
the `modal-footer` should be outside the `modal-body`
```suggestion
        </div>
        <div class="modal-footer">
          <cd-form-button-panel (submitActionEvent)="onSubmit()"
                                [submitText]="actionLabels.SUBMIT"
                                [form]="configForm"></cd-form-button-panel>
        </div>
```